src/ao-profile/profile.lua

```diff
@@ -9,22 +9,27 @@ local json = require('json')
 --   ProfileImage
 --   DateCreated
 --   DateUpdated
+--	 Version
 -- }
+if not CurrentProfileVersion then CurrentProfileVersion = 0.1 end
```

Comment:
Is the intention that when we eval new code, the new code knows what version it is? Then maybe we don't want to check `if not CurrentProfileVersion`? not 100% clear on this.